Tools/autotest/sim_vehicle.py

```diff
@@ -350,6 +350,24 @@ def do_build_waf(opts, frame_options):
     os.chdir(old_dir)
 
 
+def do_build_parameters(vehicledir):
+    # build succeeded
+    # now build parameters
+    progress("Building fresh parameter descriptions")
+    old_dir = os.getcwd()
+    param_metadata_dir = os.path.join(find_root_dir(),
+                                      "Tools/autotest/param_metadata")
+    os.chdir(param_metadata_dir)
+    cmd_param_build = ["python",
+                       "./param_parse.py",
+                       '--vehicle',
+                       os.path.basename(vehicledir)]
+
+    run_cmd_blocking("Building fresh params", cmd_param_build)
```

Comment:
Shouldn't we check that it didn't give any error?

Reply:
Done.  Nobody else in there does, 'though :-)